api.swagger.yaml

```diff
@@ -661,6 +663,45 @@ paths:
           description: The supplied UUID was malformed.
         '404':
           description: The UUID does not correspond to an existing user.
+  '/users/{user_id}/activities':
+    parameters:
+      - in: path
+        name: user_id
+        description: The Invotra UUID of the user.
+        required: true
+        schema:
+          type: string
+          format: uuid
+    get:
+      tags:
+        - users
+      description: Return the user's activity feed.
+      parameters:
```

Comment:
The `/activities` endpoint has `page` parameter while `/users/{user_id}/activities` has `offset` and `limit`.
Also, the `/users/{user_id}/activities` does not have `app_feeds_only`.

Is this intentional?

Reply:
Well, it was just carried over from the previously existing endpoints that these deprecate, which already had those differences. But yeah, given that we're deprecating and replacing them, we might as well tidy up the new versions too. So I've switched `/activities` to `offset` and `limit` for consistency with everything else.

As for `app_feeds_only`, I'm not sure. Let me get back to you on that one.

Reply:
Confirmed that `/users/{user_id}/activities` should indeed have `app_feeds_only`.